lib/foreman_maintain/report.rb

```diff
@@ -0,0 +1,59 @@
+module ForemanMaintain
+  class Report < Executable
+    include Concerns::Logger
+    include Concerns::SystemHelpers
+    include Concerns::Metadata
+    include Concerns::Finders
+
+    attr_accessor :data
+
+    def sql_count(sql, column: '*', cte: '')
+      sql_as_count("COUNT(#{column})", sql, cte: cte)
+    end
+
+    def sql_as_count(selection, sql, cte: '')
```

Comment:
what does `cte` stand for?

Reply:
introduced in fd7c0c7428ddf4d40e9945d19c47e1d2e480becf , I guess @adamruzicka would know the best :-)

Reply:
[common table expression(s)](https://www.postgresql.org/docs/current/queries-with.html)

Reply:
TIL, thanks

Reply:
And will forget. I hate to be pedantic.. actually I don;t hate it, maybe hate it for you :) Can we spell that out somehow please? The meaning of `cte` will be lost when someone has to read this later in life.